### ISSUE TITLE
SUMIFS function

### ISSUE THREAD

**Issue by ycdaskin** [unvell/ReoGrid#152]
Hello,

I wrote a custom function like "SUMIFS" in excel. It works as expected, but when i save the file it cannot be opened in excel. I think the main reason is excel uses semicolon as seperator between arguments but reogrid uses comma. Is there a way to save custom functions excel compatible?

Sample usage of formula:
=SUMIFS(E1:E8,A1:A8,"True",B1:B8,"OK",C1:C8,"NaN",D1:D8,"GASSED")
First argument is the sum range.  The other arguments are range - criteria pairs.

Here my custom function: 
```
FormulaExtension.CustomFunctions["SUMIFS"] = (cell, args) =>
            {
                if (args.Length == 0)
                {
                    return null;
                }
                try
                {
                    double result = 0;
                    var sumRange = args[0].ToString();
                    var rangeLetters = new List<string>();
                    var criterias = new List<string>();
                    for (int i = 1; i < args.Length; i+=2)
                    {
                        rangeLetters.Add(args[i].ToString()[0].ToString());
                    }
                    for (int i = 2; i < args.Length; i+=2)
                    {
                        criterias.Add(args[i].ToString());
                    }
                    bool criteriaTest = true;
                    cell.Worksheet.IterateCells(sumRange, (row, col, c) =>
                    {
                        for (int i = 0; i < criterias.Count; i++)
                        {
                            criteriaTest = (string)cell.Worksheet.Cells[rangeLetters[i] + (row + 1)].Data == criterias[i];
                            if (!criteriaTest)
                            {
                                break;
                            }
                        }
                        result = criteriaTest ? result + (double)c.Data : result;
                        criteriaTest = true;
                        return true;
                    });
                    return result;
                }
                catch (Exception ex)
                {
                    return null;
                }
            };
```

**Comment by jingwood** [comment]
Nice customize function implementation!

It seems Excel uses the [language culture settings](https://lockone.wordpress.com/2015/08/06/excel-now-using-semicolons-instead-of-commas-in-separating-formulas-fix-it-now/) to decide the parameter separator, I think we can also do it. I have [made a change](https://github.com/unvell/ReoGrid/commit/fcf79a666cfb33002648c2d0050439d186c5d4fe) for this. Can you please try from the latest source code?

**Comment by ycdaskin** [comment]
Thank you for your response. I tried last source code. Unfortunately no seperator works now. I tried both way semicolon and comma but reogrid control cannot calculate the formula. I also tried other possible seperators but none of them worked. By the way my culture info is Turkish.

**Comment by jingwood** [comment]
Oh, you are trying to save a ReoGrid customize function and use it in Excel, I think it is impossible. Both ReoGrid and Excel don't support this.

**Comment by jingwood** [comment]
I think this has been fixed by #392. If you still have the problem please let me know.
